spec/haml_lint/directive_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'spec_helper'
+
+RSpec.describe HamlLint::Directive do
+  subject(:directive) { described_class.new }
+
+  describe '.from_line' do
+    let(:line) { 1 }
+
+    subject(:directive) { described_class.from_line(source, line) }
+
+    context 'for a line that is a directive' do
+      let(:source) { ' haml-lint:disable AltText, LineLength' }
```

Comment:
Would it make sense to test a few more strings? I'm thinking single linter, different whitespace, and perhaps one that has the linter misspelled?